electrumsv/tests/test_wallet.py

```diff
@@ -151,7 +152,10 @@ def check_legacy_parent_of_standard_wallet(wallet: Wallet,
         # This account was created for the test and gets the correct flag
         assert account_keystore.get_masterkey_flags() == MasterKeyFlags.ELECTRUM_SEED
     else:
-        assert account_keystore.get_parent_keystore() is None
+        migration_number = wallet._storage.get('migration')
```

Comment:
This is a workaround. You have seen some change in behaviour and added the minimum possible bypass logic so you do not see it. You need to be able to reconcile:
a) Why it was not happening before.
b) Why it is happening now.